spaceship.zsh

```diff
@@ -525,6 +533,30 @@ spaceship_hg() {
     "$SPACESHIP_HG_SUFFIX"
 }
 
+# EMBER
+# Show current version of ember, exception system.
+spaceship_ember() {
+  [[ $SPACESHIP_EMBER_SHOW == false ]] && return
+
+  # Show EMBER status only for folders w/ ember-cli-build.js files
+  [[ -f ember-cli-build.js ]] || return
+
+  local ember_version
+
+  if _exists ember version; then
```

Comment:
`_exists` helper takes only first argument, so `version` will be ignored here. Just do `_exists ember`